inspectit-oce-core/src/test/java/rocks/inspectit/oce/core/exporter/PrometheusExporterTest.java

```diff
@@ -0,0 +1,88 @@
+package rocks.inspectit.oce.core.exporter;
+
+import ch.qos.logback.classic.Level;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.conn.ConnectTimeoutException;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.annotation.DirtiesContext;
+import rocks.inspectit.oce.core.SpringTestBase;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class PrometheusExporterTest extends SpringTestBase {
```

Comment:
let's call this `PrometheusExporterIntTest` and have a clear separation between unit and integration tests?